pkg/ccl/backupccl/backup_planning.go

```diff
@@ -1575,104 +1548,174 @@ func getBackupDetailAndManifest(
 
 	prevBackups, encryptionOptions, memSize, err := fetchPreviousBackups(ctx, &mem, user,
```

Comment:
It makes me slightly uneasy that the error handling for this error has strayed further away from the assignment. Can we move the error check and defer back or am I missing something?

Reply:
Yup, just an oversight. Thanks for catching!